vgpu-manager/rhel8/nvidia-driver

```diff
@@ -37,6 +37,21 @@ _create_dev_char_directory() {
 	fi
 }
 
+_set_fw_search_path() {
+    local nv_fw_search_path="$RUN_DIR/driver/lib/firmware"
+    local set_fw_path="true"
```

Comment:
Are we using this variable?

Reply:
We are not. Good catch. I have removed this.